packages/dht/protos/DhtRpc.proto

```diff
@@ -177,7 +177,15 @@ message RouteMessageWrapper {
 
 message RouteMessageAck {
   string requestId = 1;
-  string error = 2;
+  optional RouteMessageError error = 2;
+}
+
+enum RouteMessageError {
+  NO_TARGETS = 0;
```

Comment:
Does each message has one target? I.e. should this be `NO_TARGET`?

Reply:
Yes the message has a single target but from the perspective of a router it checks if it knows anyone it can propagate the message to. The source node will also route the message to two peers initially